### PR TITLE
Polygon validators active set alerts

### PR DESCRIPTION
**Pull request by madlabman**

### CODE REVIEW

File: lido-on-polygon/src/agent-node-operators.ts [hunk 1]
```diff
@@ -157,3 +163,83 @@ async function handleNodeOperatorsNftOwners(
...
+    .forEach((r) => {
+      addToReported(r);
+      findings.push(
+        Finding.fromObject({
+          name: "🚨 Lido validator is not in the active set",
+          description: `Lido validator ${LIDO_VALIDATORS_IDS[r.vId]} of NFT ${
+            r.vId
+          } is not in the active set`,
+          alertId: "LIDO-VALIDATOR-NOT-IN-ACTIVE-SET",
+          severity: FindingSeverity.High,
```
Comment (dgusakov) [inline lido-on-polygon/src/agent-node-operators.ts:223]:
```suggestion
          severity: FindingSeverity.Critical,
```

File: lido-on-polygon/src/agent-node-operators.ts [hunk 2]
```diff
@@ -157,3 +163,83 @@ async function handleNodeOperatorsNftOwners(
...
+        isValidator: request[0], // why its an array?
+        vId,
+      };
+    })
+  );
+
+  // first time or every 100 blocks i.e. ~ 20 min otherwise
+  const mayFire = (r: IsValidatorResult) =>
+    !lidoValidatorGoesInactiveReport.has(r.vId) ||
+    blockEvent.blockNumber % 100 == 0;
```
Comment (dgusakov) [inline lido-on-polygon/src/agent-node-operators.ts:209]: Move value to consts + I don't think it is passible to do something with the situation in 20 mins. Let it be once per 24h

File: lido-on-polygon/src/agent-node-operators.ts [hunk 3]
```diff
@@ -157,3 +163,83 @@ async function handleNodeOperatorsNftOwners(
...
+        })
+      );
+    });
+
+  rows.filter(isActive).forEach((r) => {
+    const reported = popFromReported(r);
+    if (reported) {
+      findings.push(
+        Finding.fromObject({
+          name: "🚨 Lido validator is back in the active set",
```
Comment (dgusakov) [inline lido-on-polygon/src/agent-node-operators.ts:234]:
```suggestion
          name: "✅ Lido validator is back in the active set",
```

**Review by dgusakov — CHANGES_REQUESTED** [review]

File: lido-on-polygon/src/agent-node-operators.ts (hunk 2 above)
Comment (madlabman) [reply lido-on-polygon/src/agent-node-operators.ts]: Fixed.

**Review by dgusakov — DISMISSED** [review]

**Review by dgusakov — APPROVED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Pull request merged**